### PR TITLE
Configuration: Enabling Desktop Icons

### PR DESCRIPTION
**Pull request by NeilWB**
Added GNOME & MATE sections and tweaked Budgie a bit.  Updated the Nautilus screenshot as well.

As per #44

Hopefully everything is in order!

### CODE REVIEW

File: configuration/enabling-desktop-icons/en.md [hunk 1]
```diff
@@ -3,12 +3,34 @@ title = "Enabling Desktop Icons"
 +++
 # Enabling Desktop Icons
 
-You can enable desktop icons by opening Raven, clicking the cog wheel for Budgie Settings, then toggling Desktop Icons on under the Background section.
+You can enable or disable desktop icons easily depending on your personal preference.
+
+## Budgie
+
+With Budgie, start by opening Raven, either by clicking on its icon in the far right of the system tray or using the keyboard shortcut `Windows Key + A`.  Next, click the cog icon for Budgie Settings and then toggle "Desktop Icons" on under the Background section.
```
Comment (ikeydoherty) [inline configuration/enabling-desktop-icons/en.md]: Windows is a trademark so to refer to it as the windows key would involve marking it as a trademark. Should be referred to as the <kbd>Super</kbd> key

File: configuration/enabling-desktop-icons/en.md [hunk 2]
```diff
@@ -3,12 +3,34 @@ title = "Enabling Desktop Icons"
...
+
+{{< altimg "nautilus-desktop-folder.jpg" "help-center/configuration/enabling-desktop-icons/" >}}
+
+## MATE
+
+MATE has desktop icons enabled by default.  If you wish to disable or re-enable them, open the Brisk Menu, click on "Preferences" and then on "MATE Tweak".  You can then toggle them on and off by clicking "Show Desktop Icons" under the Desktop section.
+
+{{< altimg "mate-tweak-desktop.jpg" "help-center/configuration/enabling-desktop-icons/" >}}
+
+You may now add things to your Desktop by opening the Caja file manager and then moving the files you want to the Desktop folder (as shown in the image below).
```
Comment (ikeydoherty) [inline configuration/enabling-desktop-icons/en.md]: If you have desktop icons enabled you don't need to use the file manager, as the desktop _is the file manager_. You can right click and drag etc

File: configuration/enabling-desktop-icons/en.md (hunk 1 above)
Comment (NeilWB) [inline configuration/enabling-desktop-icons/en.md]: Being a Mac user I was going to write the Command key!  For what it's worth, I went with the shortcuts listed [here](https://solus-project.com/articles/budgie/shortcuts/en/).

File: configuration/enabling-desktop-icons/en.md (hunk 2 above)
Comment (NeilWB) [inline configuration/enabling-desktop-icons/en.md]: Copy that, will tweak it.

**Comment by NeilWB** [comment]
Removed Windows. 

Have tweaked the wording to make it clear there are alternatives to using "Files" or "Caja".  I kept the screenshots as the Desktop folder doesn't show until the icons are enabled hence it shows people what it should look like after they've turned them on.

**Pull request merged**